retrievalmarket/impl/integration_test.go

```diff
@@ -275,7 +280,7 @@ func TestClientCanMakeDealWithProvider(t *testing.T) {
 		{name: "multi-block file retrieval succeeds",
 			filename:    "lorem.txt",
 			filesize:    19000,
-			voucherAmts: []abi.TokenAmount{abi.NewTokenAmount(10136000), abi.NewTokenAmount(19920000)},
+			voucherAmts: []abi.TokenAmount{abi.NewTokenAmount(10174000), abi.NewTokenAmount(19958000)},
```

Comment:
I must be dense but I've read through this multiple times and still cant figure out why the vouchers changed. It appears (I think) that everything went up by exactly 38000? Which happens to be 2x the filesize? I'd like to understand why.

Reply:
Good question -- @dirkmc could you investigate this? It does sound like an unexpected functional change.

Reply:
To be honest I'm not sure either. This is the commit where the change was made:
https://github.com/filecoin-project/go-fil-markets/commit/cf7e2a9ecb1cda8ae5331c8a248b3a7f3d54ac97
But I'm not sure why the amounts are different.

Reply:
@raulk please take a look 👁️👁️

Reply:
Following investigation, this change is correct. We modified the `DagBuilderParams` to match those used in `lotus client import` for a more realistic test, and that leads to a heavier DAG, thus higher cost.

If I revert that change, the voucher amounts revert to the previous ones.

Conclusion: side-effect of the test changing (for the better).

![image](https://user-images.githubusercontent.com/1101242/129074108-a2b1bf0b-e82e-476a-bf03-2be05e65f7f3.png)

https://github.com/filecoin-project/go-fil-markets/commit/cf7e2a9ecb1cda8ae5331c8a248b3a7f3d54ac97#diff-26f3d1665dab327881c8abd11dfa4b1e249f730bddfefdec7363fe08290148fdR113-R122

Reply:
@raulk is correct here.